cmd/cycloid/terracost/cmd.go

```diff
@@ -0,0 +1,24 @@
+package terracost
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/cycloidio/cycloid-cli/cmd/cycloid/common"
+)
+
+// NewCommands returns an implementation of Terracost commands
+func NewCommands() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "terracost",
```

Comment:
Maybe double check for the public branding name we will give to this feature/option.
Maybe it will keep the infraCost name or cost-estimation ?

Reply:
`terracost` has been decided internally :)